types/mongodb/index.d.ts

```diff
@@ -2,6 +2,7 @@
 // Project: https://github.com/mongodb/node-mongodb-native/tree/2.2
 // Definitions by: Federico Caselli <https://github.com/CaselIT>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3'
```

Comment:
:expressionless: Ahhh.. such a small feature and already ruines all travis checking.